intro-HPC/ch_quick_reference_guide.tex

```diff
@@ -1,52 +1,63 @@
 \chapter{\hpc Quick Reference Guide}
 \label{ch:quick-reference-guide}
 
+Remember to subsitute the usernames, login nodes, file names, \ldots for your own.
+
 \begin{tabular}{|l|l|} \hline
 \multicolumn{2}{|c|}{\strong{Login}} \\ \hline
-Login             & ssh $<$vsc-account$>$@\loginnode \\ \hline
-Where am I?       & hostname \\ \hline
-Copy to \hpc      & scp foo.txt $<$vsc-account$>$@\loginnode: \\ \hline
-Copy from \hpc    & scp $<$vsc-account$>$@\loginnode:foo.txt . \\ \hline
-Setup ftp session & sftp $<$vsc-account$>$@\loginnode \\ \hline
+Login             & \texttt{ssh \userid{}@\loginnode{}}\\ \hline
+Where am I?       & \texttt{hostname} \\ \hline
+Copy to \hpc      & \texttt{scp foo.txt \userid{}@\loginnode{}:} \\ \hline
+Copy from \hpc    & \texttt{scp \userid{}@\loginnode{}:foo.txt .} \\ \hline
+Setup ftp session & \texttt{sftp \userid{}@\loginnode{}} \\ \hline
 \end{tabular}
 
 \begin{tabular}{|l|l|} \hline
 \multicolumn{2}{|c|}{\strong{Modules}} \\ \hline
-List all available modules & module av \\ \hline
-List loaded modules        & module list \\ \hline
-Load module                & module load $<$name$>$ \\ \hline
-Unload module              & module unload $<$name$>$ \\ \hline
-Unload all modules         & module purge \\ \hline
-Help on use of module      & module help \\ \hline
+List all available modules & \texttt{module avail} \\ \hline
+List loaded modules        & \texttt{module list} \\ \hline
+Load module                & \texttt{module load example} \\ \hline
+Unload module              & \texttt{module unloade example} \\ \hline
+Unload all modules         & \texttt{module purge} \\ \hline
+Help on use of module      & \texttt{module help} \\ \hline
 \end{tabular}
 
 \begin{tabular}{|l|l|} \hline
 \multicolumn{2}{|c|}{\strong{Jobs}} \\ \hline
-Submit Job              & qsub $<$script.pbs$>$ \\ \hline
-Status of the job       & qstat $<$jobid$>$ \\ \hline
-Possible start time (not available everywhere)    & showstart $<$jobid$>$ \\ \hline
-Check job (not available everywhere)              & checkjob $<$jobid$>$ \\ \hline
-Show compute node       & qstat -n $<$jobid$>$ \\ \hline
-Delete job              & qdel $<$jobid$>$ \\ \hline
-Status of all your jobs & qstat \\ \hline
-Show all jobs on  queue (not available everywhere) & showq \\ \hline
+Submit Job              & \texttt{qsub script.pbs} \\ \hline
+Status of the job       & \texttt{qstat jobid} \\ \hline
```

Comment:
I think it would be clearer if you use `qstat 12345` and change the description to `Status of job with ID 12345`?

Same below